contracts/NFTStorefront.cdc

```diff
@@ -22,17 +22,54 @@ import NonFungibleToken from 0xNONFUNGIBLETOKEN
 // and list items of interest.
 //
 pub contract NFTStorefront {
+    // NFTStorefrontInitialized
+    // This contract has been deployed.
+    // Event consumers can now expect events from this contract.
+    //
+    pub event NFTStorefrontInitialized()
+
+    // StorefrontInitialied
+    // A Storefront resource has been created.
+    // Event consumers can now expect events from this Storefront.
+    // Note that we do not specify an address: we cannot and should not.
+    // Created resources do not have an owner address, and may be moved
+    // after creation in ways we cannot check.
+    // SaleOfferAvailable events can be used to determine the address
+    // of the owner of the Storefront (...its location) at the time of
+    // the offer but only at that precise moment in that precise transaction.
+    // If the offerer moves the Storefront while the offer is valid, that
+    // is on them.
+    //
+    pub event StorefrontInitialied(storefrontResourceID: UInt64)
+
+    // StorefrontCreated
```

Comment:
```suggestion
    // StorefrontDestroyed
```

Reply:
Thank you for catching this. I've updated it.